### DISCUSSION TITLE
如何在点击 DataGrid 列进行排序时保持当前分组顺序？

### DISCUSSION THREAD

**Discussion by CodingOctocat** [BYJRK/DotNet-Discussions#191] [WPF 开发]
启用 DataGrid 的排序功能时，当点击列标题，可以对数据进行排序，但是如果同时启用分组功能，点击排序时，分组顺序会被打乱，我希望点击列标题时，排序是针对各个组内排序。

> 尽量以 MVVM 方式实现。

我尝试过：[How to force DataGrid group order in WPF?](https://stackoverflow.com/questions/10353535/how-to-force-datagrid-group-order-in-wpf)，但是数据无法正常显示。

**Comment by CodeWangtong** [comment]
个人建议点击排序binding command 
后端根据 分页index 和排序方式 OrderBy写筛选表达式

**Reply by CodingOctocat** [reply]
暂时禁用排序解决问题😎
